swift_browser_ui_frontend/src/entries/main.js

```diff
@@ -135,8 +141,20 @@ new Vue({
           );
         }
       });
+    delay(
+      this.containerSyncWrapper,
+      5000,
+    );
   },
   methods: {
+    containerSyncWrapper: function () {
+      syncContainerACLs(
+        this.$store.state.client,
+        this.$store.state.active.id,
+      ).then(() => {
+        console.log("ACLs have been updated.");
```

Comment:
Did you leave this intentionally?

Reply:
Somewhat intentionally – I'd like to flag the process somewhere, but a toast seems like overdoing it. It's true that console messages in production software are a bit unprofessional, though, so there's probably a better option somewhere. Maybe some sort of a loading indicator for background information could be useful for the UI.

I'll ditch the rest of the console messages.

Reply:
Removed the console messages, popped open an issue on adding a better loading indicator for the UI since we have begun to fetch more dynamic content in the background. Issue together with the discussion on it can be found in #425